### PR TITLE
Update korean strings to follow changes in 4.5.0

### PR DESCRIPTION
**Pull request by cozyplanes**
need to add a bit more, will finish by wednesday

### CODE REVIEW

**Comment by yakov116** [comment]
Travis is making issues now, @k0shk0sh im merging

**Comment by yakov116** [comment]
@cozyplanes just saw your message, will wait to merge

**Comment by yakov116** [comment]
@cozyplanes 
https://github.com/cozyplanes/FastHub/blob/206bf1899afe90097a4e421a1cadfb63d228dffb/app/src/main/res/values-ko/strings.xml#L467
https://github.com/cozyplanes/FastHub/blob/206bf1899afe90097a4e421a1cadfb63d228dffb/app/src/main/res/values-ko/strings.xml#L437

Are duplicates

**Comment by k0shk0sh** [comment]
```
Caused by: com.android.build.gradle.tasks.ResourceException: /home/travis/build/k0shk0sh/FastHub/app/src/main/res/values-ko/strings.xml: Error: Found item String/faq more than one time
```

@cozyplanes please do not open something and say wait until x days, instead make your commits local and only push on when you done.

**Comment by k0shk0sh** [comment]
Its Friday by your time

**Comment by cozyplanes** [comment]
finished

sorry for the delay
we had some major power outage couple of days before

**Comment by dedepete** [comment]
Please, remove the duplicate mentioned above.

**Comment by cozyplanes** [comment]
fixed duplicate, thanks for the mention

File: app/src/main/res/values-ko/strings.xml [hunk 1]
```diff
@@ -434,7 +434,6 @@
     <string name="too_large_changes">변경사항이 너무 많습니다. 브라우저로 보세요</string>
     <string name="project">프로젝트</string>
     <string name="please_read">꼭 읽어주세요</string>
-    <string name="faq">자주하는 질문</string>
     <string name="comments_added_successfully">댓글을 성공적으로 추가하였습니다.</string>
```
Comment (k0shk0sh) [inline app/src/main/res/values-ko/strings.xml]: here

File: app/src/main/res/values-ko/strings.xml [hunk 2]
```diff
@@ -450,18 +449,25 @@
...
-        <li>플레이 스토어에서 FastHub 별점 매기기</li>
+        <li>GitHub 닉네임;</li>
+        <li>Medium같은 곳에 FastHub에 대한 홍보글을 올려주세요;</li>
+        <li>자신이 글을 썼다는 것을 증명해 주세요.</li>
         </ul>
         <h5>위 조건을 만족시킨 후 본 앱의 개발자인 Kosh Sergani에게 이메일을 보내면 검토후 PROMO CODE를 보내 드립니다.</h5>\n
         <h5>본 한국어 번역자는 이 앱의 개발에 참여하지 않았습니다. 따라서 기술적 문의는 이 앱의 개발자인 Kosh Sergani에게 해주시기 바랍니다. 본 번역이 앱 사용에 도움이 되셨기를 바랍니다. 감사합니다.</h5>
 	]]></string>
+    <string name="faq">자주 하는 질문 (FAQ)</string>
+    <string name="comments_added_successfully">댓글을 성공적으로 추가하였습니다</string>
```
Comment (k0shk0sh) [inline app/src/main/res/values-ko/strings.xml]: and here

**Pull request merged**